website/js/index.js

```diff
@@ -434,7 +430,49 @@ function isEmptyDiff(value) {
     return value === null || value.length < 1;
 }
 
-// let masonryHack = null;
+// attempts to parse the given date string using a variety of formats,
+// returning the formatted result as something like '08 September 2016'.
+// just returns the input if every pattern fails to match
+function normalizeDateFieldDisplay(value) {
```

Comment:
is any of this necessary for date formatting? I know we use moment.js (search for the use of moment in js/index.js and js/Releases.js) to handle date formatting elsewhere -- it seems we should be able to make use of this library throughout the app rather than writing custom functions. Also, shout out to how frustrating date formatting is in general.